sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -220,7 +220,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     /**
      * @brief Specify the transactional md5 for the body, to be validated by the service.
      */
-    Azure::Core::Nullable<std::string> ContentMd5;
+    Azure::Core::Nullable<ContentHash> ContentMd5;
```

Comment:
What if in the future this API supports CRC64, are you going to rename this varaible? It's going to be a breaking change.